calico-cloud/network-policy/policy-tiers/tiered-policy.mdx

```diff
@@ -64,16 +64,17 @@ The default tier is where:
 
 ## System tiers
 
-System tiers are added during installation and are hidden by default.
+System tiers are added during installation and are hidden by default. Create all of your tiers after `allow-tigera` and `tigera-security`.
 
-- **allow-tigera** - contains component policies
-- **tigera-security** - contains threat protection policies
+- **allow-tigera** tier contains policies to secure {{prodname}} components and are controlled by the Tigera Operator. These policies should not be edited, and the tier should not be moved. Inadvertent changes are automatically reverted by the Operator to ensure your cluster is always protected.
+
+   :::warning
 
-:::note
+   Although it is possible to change the behavior of the `allow-tigera` using adjacent tiers, it is not a trivial task. You can break critical cluster traffic and impact the operation of {{prodname}}. To prevent loss of cluster services, see [Change allow-tigera tier behavior](allow-tigera.mdx), and contact Support for help.
```

Comment:
"it is not a trivial task." makes it sounds merely difficult. Perhaps better to describe it as risky or dangerous? Just a thought.

Reply:
It is both difficult and risky to get it right.